packages/core/admin/webpack.config.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
 const path = require('path');
+const fse = require('fs-extra');
 const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ForkTsCheckerPlugin = require('fork-ts-checker-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
I think you can also remove `terser-webpack-plugin` from the `package.json`?

Reply:
Done in [df6fa2f](https://github.com/strapi/strapi/pull/12882/commits/df6fa2f8b6a3794056f7e30c5a50a30821f183df)